packages/build/src/operators/developmentOperator.ts

```diff
@@ -0,0 +1,60 @@
+import * as E from '../services'
+import { LazyPageOperator } from './lazyPageOperator'
+import { Operator, Wiring } from './operator'
+import * as O from './operator'
+
+export class DevOperator extends Operator {
+  wiring: Wiring[] = [
+    { once: O.StartEvent, emit: E.PrepareEvent },
+    { once: E.PreparedEvent, emit: E.CompileEvent },
+    { on: E.CompiledEvent, emit: E.GenerateCodeEvent },
+    { once: E.GeneratedCodeEvent, emit: E.BundleVendorEvent },
+    { once: E.GeneratedCodeEvent, emit: E.StartBackendEvent },
+    { on: E.BundledPageEvent, emit: E.PageBundleEmitted },
+  ]
+  startupTimer?: [number, number]
+  config = {
+    compatible: false,
+    fast: false,
+    optimized: false,
+  }
+
+  constructor(config: any) {
+    super()
+    this.applyConfig(config)
+    this.on(O.StartEvent, this.startTimer)
+
+    this.addDevServices()
+    this.setupWiring()
+    this.addLazyPageOperator()
+
+    this.on(E.StartBackendEvent, this.stopTimer)
+  }
+
+  applyConfig(config: any) {
+    const mapped = {
+      compatible: config.compatible,
+      fast: !config.typeCheck,
```

Comment:
no config has other keys (typeCheck, optimizeBundle)
used in ./bin/build|server js so types would arguable be just noise

We keep the default ones in case there are keys missing in the given config, just some defensive programming without a lot of thought